app/_data/docs_nav_kuma_dev.yml

```diff
@@ -486,6 +486,8 @@ items:
         url: /guides/migration-to-the-new-policies/
       - text: Progressively rolling in strict MTLS
         url: /guides/progressively-rolling-in-strict-mtls/
+      - text: Producer and consumer policies
+        url: /guides/consumer-producer-policies
```

Comment:
You now need to add it to 2.9 sidebar too

Reply:
done